### ISSUE TITLE
I think we should use "euclidean_distance" rather than "euclidean_distance2"

### ISSUE THREAD

**Issue by BAEKSEONGYUN** [NSMARTS/Face-Recognition#6]
```python
def euclidean_distance(vectors):
    # x, y = vectors
    x = vectors[0]
    y = vectors[1]
    sum_square = tf.math.reduce_sum(
        tf.math.square(x - y), axis=1, keepdims=True)
    return tf.math.sqrt(tf.math.maximum(sum_square, tf.keras.backend.epsilon()))
```

```python
def euclidean_distance2(vectors):
    vector1, vector2 = vectors
    sum_square = K.sum(K.square(vector1 - vector2), axis=1, keepdims=True)
    return K.sqrt(K.maximum(sum_square, K.epsilon()))
```
The two functions calculate essentially the same Euclidean distance. The resulting values for both functions will be almost identical.

As TensorFlow integrated Keras, the boundaries between the two libraries were blurred. So choosing between the two functions won't make much difference.

However, in order to maintain the consistency of the code and take advantage of the characteristics of a particular framework, the first function is recommended if TensorFlow is primarily used.
